packages/components/src/font-size-picker/utils.js

```diff
@@ -59,21 +57,19 @@ export function isSimpleCssValue( value ) {
  * @param {boolean}  useSelectControl               Whether to use a select control.
  * @param {Object[]} optionsArray                   Array of available font sizes objects.
  * @param {*}        disableCustomFontSizes         Flag that indicates if custom font sizes are disabled.
- * @param {boolean}  optionsContainComplexCssValues Whether font sizes contain at least one complex css value(clamp, var, etc..).
  * @return {Object[]|null} Array of font sizes in proper format for the used control.
  */
```

Comment:
Should we tweak `disableCustomFontSizes` type here to be `boolean` as per the readme? Also, I'm getting an eslint warning for "JSDoc block lines to be aligned".